src/covid19sim/simulator.py

```diff
@@ -185,29 +184,33 @@ def __init__(self, env, city, name, age, rng, has_app, infection_timestamp, hous
-        self.asymptomatic_infection_ratio = ASYMPTOMATIC_INFECTION_RATIO if self.is_asymptomatic else 0.0 # draw a beta with the distribution in documents
+        self.is_asymptomatic = self.rng.rand() < self.conf.get("BASELINE_P_ASYMPTOMATIC") - (self.age - 50) * 0.5 / 100 # e.g. 70: baseline-0.1, 20: baseline+0.15
+        self.asymptomatic_infection_ratio = (
+            self.conf.get("ASYMPTOMATIC_INFECTION_RATIO")
+            if self.is_asymptomatic
+            else 0.0
+        ) # draw a beta with the distribution in documents
 
         # Indicates whether this person will show severe signs of illness.
-        self.cold_timestamp = self.env.timestamp if self.rng.random() < P_COLD else None
-        self.flu_timestamp = self.env.timestamp if self.rng.random() < P_FLU else None # different from asymptomatic
-        self.allergy_timestamp = self.env.timestamp if self.rng.random() < P_HAS_ALLERGIES_TODAY else None
+        self.cold_timestamp = self.env.timestamp if self.rng.random() < self.conf.get("P_COLD") else None
+        self.flu_timestamp = self.env.timestamp if self.rng.random() < self.conf.get("P_FLU") else None # different from asymptomatic
+        self.allergy_timestamp = self.env.timestamp if self.rng.random() < self.conf.get("P_HAS_ALLERGIES_TODAY") else None
         self.can_get_really_sick = _get_get_really_sick(self.age, self.sex, self.rng)
         self.can_get_extremely_sick = self.can_get_really_sick and self.rng.random() >= 0.7 # &severe; 30% of severe cases need ICU
-        self.never_recovers = self.rng.random() <= P_NEVER_RECOVERS[min(math.floor(self.age/10),8)]
+        self.never_recovers = self.rng.random() <= self.conf.get("P_NEVER_RECOVERS")[min(math.floor(self.age/10),8)]
         self.obs_hospitalized = False
         self.obs_in_icu = False
 
         # possibly initialized as infected
         self.recovered_timestamp = datetime.datetime.min
         self.is_immune = False # different from asymptomatic
         self.viral_load_plateau_height, self.viral_load_plateau_start, self.viral_load_plateau_end, self.viral_load_recovered = None,None,None,None
-        self.infectiousness_onset_days = None # 1 + self.rng.normal(loc=INFECTIOUSNESS_ONSET_DAYS_AVG, scale=INFECTIOUSNESS_ONSET_DAYS_STD)
-        self.incubation_days = None # self.infectiousness_onset_days + self.viral_load_plateau_start + self.rng.normal(loc=SYMPTOM_ONSET_WRT_VIRAL_LOAD_PEAK_AVG, scale=SYMPTOM_ONSET_WRT_VIRAL_LOAD_PEAK_STD)
+        self.infectiousness_onset_days = None #  (/!\ self.conf if uncomment) 1 + self.rng.normal(loc=INFECTIOUSNESS_ONSET_DAYS_AVG, scale=INFECTIOUSNESS_ONSET_DAYS_STD)
+        self.incubation_days = None # (/!\ self.conf if uncomment) self.infectiousness_onset_days + self.viral_load_plateau_start + self.rng.normal(loc=SYMPTOM_ONSET_WRT_VIRAL_LOAD_PEAK_AVG, scale=SYMPTOM_ONSET_WRT_VIRAL_LOAD_PEAK_STD)
```

Comment:
what is this warning?

Reply:
this means that I didn't know if this comment was ever going to be uncommented. If it is uncommented, it's a warning that global variables don't exist anymore and whoever's uncommenting should get those variable from `self.conf`